core/src/io/anuke/mindustry/editor/MapRenderer.java

```diff
@@ -97,59 +101,91 @@ public void updateAll(){
     }
 
     private void render(int wx, int wy){
-        int x = wx / chunkSize, y = wy / chunkSize;
-        IndexedRenderer mesh = chunks[x][y];
-        Tile tile = editor.tiles()[wx][wy];
+        final Point windowPoint = new Point(wx, wy);
+        final Point gridPoint = new Point(wx/chunkSize, wy/chunkSize);
```

Comment:
Don't mark variables as `final` unless they're static/instance variables. It doesn't increase performance, and IntelliJ marks non-implicitly-final variables with different colors anyway.